lib/membrane_mp3_mad_plugin/decoder.ex

```diff
@@ -74,21 +66,21 @@ defmodule Membrane.MP3.MAD.Decoder do
         {:ok, {buffer, Enum.reverse(acc)}}
 
       {:error, {:recoverable, bytes_to_skip}} ->
-        warn("Skipping malformed frame (#{bytes_to_skip} bytes)")
+        Logger.warn("Skipping malformed frame (#{bytes_to_skip} bytes)")
         <<_used::binary-size(bytes_to_skip), new_buffer::binary>> = buffer
 
         case acc do
           [{:event, _} | _] ->
```

Comment:
seems like an unnamed underscore, shouldn't credo complain?

Reply:
Well, it complained about line 86, I'm not sure why this is ignored, but it is :shrug: